### PR TITLE
[BLOG] Add packaging_painpoints_in_2021

### PR DESCRIPTION
**Pull request by MarsBarLee**
Link to [blog post, successful build](https://labs-git-packagingpainpointsin2021-quansight.vercel.app/blog/packaging_painpoints_in_2021).

## Links to other Quansight blogs:
There is no need to check for Blog2Blog links due to updated website build. 

## Text styling

- [x] The blog is written with [plain language](https://www.plainlanguage.gov/guidelines/) (where relevant).
- [x] If there are headers, they use the proper header tags in order to do so (with only one level-one header).
- [x] All links describe where they link to (for example, check the [Quansight labs website](https://labs.quansight.org/)).
- [x] Any kind of styling that the author uses (for example, bold for emphasis) is consistent throughout the blog.

## Non-text contents

- [x] All content is represented as text (for example, images need alt text and videos need captions or descriptive transcripts).
- [x] If there are emojis, there are not more than three in a row.
- [x] Don't use [flashing gifs or videos](https://www.w3.org/TR/UNDERSTANDING-WCAG20/seizure-does-not-violate.html).
- [x] If it were to be read as plain text, the blog still makes sense and no information is missing.

### CODE REVIEW

File: apps/labs/posts/packaging_painpoints_in_2021.md [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+
+- Why oh why are defaults and conda-forge still not using a 100% compatible toolchain??
+
+_RG: This would be so helpful. Not being able to mix `defaults` and `conda-forge`, and users doing it anyway because they need packages from both, is a long-standing issue._
+
+- Conda solves in CI often get really bogged down
+    - Also conda is never the default in CI so you need to do quite a bit of manipulation on some platforms
+- Confusion in the community over the new Anaconda ToS
+
+_RG: Anaconda has done the right thing here imho: make large institutional users pay if they use the `defaults` channel. Packaging is hard and labour-intensive, and having a sustainable revenue stream to offset the cost of packaging difficult libraries, maintaining core tooling like `conda-build`, and serving packages to millions of users is important. Conda-forge and all other channels are still completely free. Unfortunately Anaconda communicated that very poorly, which led to unnecessary confusion. See [this blog posts from the conda-forge team](https://conda-forge.org/blog/posts/January 24, 2021
```
Comment (noatamir) [inline apps/labs/posts/packaging_painpoints_in_2021.md:130]: 1. adding a closed bracket to fix md linking.
2. conda-forge has changed their website, so this link needs updating. @rgommers can you please have a look here and identify the blog you were referring to? 
https://conda-forge.org/blog/

```suggestion
_RG: Anaconda has done the right thing here imho: make large institutional users pay if they use the `defaults` channel. Packaging is hard and labour-intensive, and having a sustainable revenue stream to offset the cost of packaging difficult libraries, maintaining core tooling like `conda-build`, and serving packages to millions of users is important. Conda-forge and all other channels are still completely free. Unfortunately Anaconda communicated that very poorly, which led to unnecessary confusion. See [this blog posts from the conda-forge team](https://conda-forge.org/blog/posts/January 24, 2021)
```

File: apps/labs/posts/packaging_painpoints_in_2021.md [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+_RG: True. For doing serious scientific computing, data science and ML type work, you often need the combination, using `conda` for the core libraries, and filling in the gaps with `pip` for packages that are not packaged for conda or installing things from source._
+
+
+## Topic 4: There's always good news too
+
+- [The death of `easy_install`](https://github.com/pypa/setuptools/pull/2544)!!!
+- Mambaforge
+- macos-arm64 support in conda-forge
+
+_RG: It'll be a while before we get wheels for core PyData packages, but conda-forge already has support, see [this blog post](https://conda-forge.org/blog/posts/January 24, 2021
```
Comment (noatamir) [inline apps/labs/posts/packaging_painpoints_in_2021.md:164]: @rgommers same thing here: https://conda-forge.org/blog/
```suggestion
_RG: It'll be a while before we get wheels for core PyData packages, but conda-forge already has support, see [this blog post](https://conda-forge.org/blog/posts/January 24, 2021)
```

File: apps/labs/posts/packaging_painpoints_in_2021.md (hunk 1 above)
Comment (rgommers) [reply apps/labs/posts/packaging_painpoints_in_2021.md:130]: https://conda-forge.org/blog/posts/2020-11-20-anaconda-tos/

File: apps/labs/posts/packaging_painpoints_in_2021.md (hunk 2 above)
Comment (rgommers) [reply apps/labs/posts/packaging_painpoints_in_2021.md:164]: Hmm, very odd. It's https://conda-forge.org/blog/posts/2020-10-29-macos-arm64/

File: apps/labs/posts/packaging_painpoints_in_2021.md [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+- Not having to think about problems around Fortran on Windows ever again.
+
+While I'm at it, let me also link to some of the most informative blog posts on the topic I know of and enjoyed reading over the years:
+
+- [Tarek Ziadé - The fate of Distutils - Pycon Summit + Packaging Sprint detailed report (2010)](https://tarekziade.wordpress.com/2010/03/03/the-fate-of-distutils-pycon-summit-packaging-sprint-detailed-report/)
+- [Travis Oliphant - Why I promote conda (2013)](http://technicaldiscovery.blogspot.com/2013/12/why-i-promote-conda.html)
+- [Jake VanderPlas - Conda: Myths and Misconceptions (2016)](https://jakevdp.github.io/blog/2016/08/25/conda-myths-and-misconceptions/)
+- [Wes McKinney - conda-forge and PyData's CentOS moment (2016)](https://wesmckinney.com/blog/conda-forge-centos-moment/)
+- [Donald Stufft - Powering the Python Package Index (2016)](https://caremad.io/posts/2016/05/powering-pypi/)
+- [Pauli Virtanen - Building Python wheels with Fortran for Windows (2017)](https://pav.iki.fi/blog/January 24, 2021
```
Comment (noatamir) [inline apps/labs/posts/packaging_painpoints_in_2021.md:222]: @rgommers this isn't online anymore. Would you prefer to replace it with this Wayback Machine link, or remove it?
```suggestion
- [Pauli Virtanen - Building Python wheels with Fortran for Windows (2017)](https://web.archive.org/web/20210616203153/https://pav.iki.fi/blog/2017-10-08/pywingfortran.html)
```

Comment (rgommers) [reply apps/labs/posts/packaging_painpoints_in_2021.md:222]: replacing with the archive.org link please

**Review by noatamir — APPROVED** [review]
LGTM. merging

**Pull request merged**